File: Kamal Sir Lab/lab_exam/matrix.py
def input_matrix(x, y, matrix):
    for i in range(x):
        matrix.append([])  # Append an empty list to represent each row
        for j in range(y):
            e = int(input(f"Enter {i}th row {j}th column element: "))
            matrix[i].append(e)  # Append the element to the respective row

def output_matrix(x, y, matrix):
    for i in range(x):
        for j in range(y):
            print(matrix[i][j], end=" ")  # Print each element of the matrix
        print(" ")  # Move to the next row

row = int(input("Enter row: "))
column = int(input("Enter column: "))
A = []  # Initialize an empty matrix
input_matrix(row, column, A)
output_matrix(row, column, A)

File: Selim Sir Lab/OOP JAVA/Assignment/2.Matrix/matrix.java
import java.util.Scanner;

public class matrix {
    public static void main(String[] args) {
        while(true)
        {
            System.out.print("What do u want: ");
            System.out.print("1.Summation: ");
            System.out.print("2.Subtraction:");
            System.out.print("1.Multiflication: ");
            System.out.print("2.Division: ");

            int op=input.nextInt()
            {
                if(op==1)
                {
                    input_matrix();
                    summation()
                }
            }

        }
    }

    // input matrix
    input_matrix(){
        System.out.print("Enter row: ");
        int r=input.nextInt();

        System.out.print("Enter column: ");
        int c=input.nextInt();

        int A[r][c];

        for(int i=0;i<r;i++)
        {
            for(int j=0;j<c;j++)
            {
                System.out.print("Enter element at position (" + i + " " + j + "): ");
                A[i][j] = input.nextInt();
            }
        }
        return A;
    }
}
